Packs/Okta/Integrations/Okta_v2/Okta_v2.py

```diff
@@ -496,6 +496,33 @@ def clear_user_sessions(self, user_id):
             resp_type='text'
         )
 
+    def get_zone(self, args, zoneID):
+        uri = f'zones/{zoneID}'
+        return self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+    def list_zones(self, args):
+        uri = f'zones'
+        return self._http_request(
+            method='GET',
+            url_suffix=uri,
+            resp_type='text'
+        )
+
+    def update_zone(self, args, zoneObject):
+        zoneID = zoneObject['id']
+        uri = f'zones/{zoneID}'
+
+        return self._http_request(
+            method='PUT',
+            url_suffix=uri,
+            resp_type='text',
+            data=json.dumps(zoneObject)
```

Comment:
```suggestion
            json_data=zoneObject
```

Reply:
Serializing the object as json is necessary.. Removing this yields {"errorCode": "E0000003", "errorSummary": "The request body was not well-formed."